step01/README.md

```diff
@@ -173,7 +171,7 @@ Since in this case we only care about the content and not the index, we can safe
 
 ## Task 03: Updating the game loop
 
-Now that we have both a `loadMaze` and a `printScreen` function, we should update the `main` function to initialize the maze and print it on the game loop. See the code below:
+Now that we have both a `loadMaze` and a `printScreen` functions, we should update the `main` function to initialize the maze and print it on the game loop. See the code below:
```

Comment:
English is not my first language,  but since you've changed function to plural you probably should remove the 'a' article before their names.